content/install_windows.md

```diff
@@ -6,54 +6,41 @@ aliases = ["/install_windows.html"]
 
 [menu]
   [menu.infra]
-    title = "Chef Infra Client on Windows"
+    title = "Windows Installation Guide"
     identifier = "chef_infra/getting_started/chef_on_windows_guide/install_windows.md Chef Infra Client on Windows"
     parent = "chef_infra/getting_started/chef_on_windows_guide"
     weight = 20
 +++
 
 [\[edit on GitHub\]](https://github.com/chef/chef-web-docs/blob/master/content/install_windows.md)
 
-## Nodes
+## Installation Methods
 
-{{% node %}}
+There are several methods available to install Chef Infra Client, depending on the needs of your organization
 
 {{% windows_install_overview %}}
 
-### Use knife windows
+### Use knife CLI
 
 {{% knife_windows_summary %}}
 
-#### Ports
+#### Necessary Ports
 
 {{% knife_windows_winrm_ports %}}
 
-### Msiexec.exe
+### Use MSI Installer
+
+A Microsoft Installer Package (MSI) is available for installing Chef Infra Client on a Microsoft Windows machine at [Chef Downloads](https://downloads.chef.io/products/infra-client?os=windows).
 
 {{% windows_msiexec %}}
 
 #### ADDLOCAL Options
 
 {{% windows_msiexec_addlocal %}}
 
-### Use MSI Installer
-
-A Microsoft Installer Package (MSI) is available for installing Chef
-Infra Client on a Microsoft Windows machine at [Chef
-Downloads](https://downloads.chef.io/)
-
-#### Run as a Scheduled Task
-
-Chef Infra Client can be run as a scheduled task. On the Microsoft
-Windows platform, a scheduled task provides more visibility,
-configurability, and reliability around log rotation and permissions, as
-compared to running Chef Infra Client as a service. A scheduled task:
+#### Running as a Scheduled Task
 
--   Prevents log rotation issues, such as filling a disk partition
--   Does not have an indeterminate status, such as a service for which
-    only the watcher is "running"
--   Prevents insufficient permissions related to service context
--   Prevents issues related to a system reboot
+On Microsoft Windows Chef Infra Client should be run periodically as a scheduled task. Scheduled tasks provides visibility, configurability, and reliability around log rotation and permissions. You can configure the Chef Infra Client to run as a scheduled task using the [chef_client_scheduled_task](/resources/chef_client_scheduled_task/) resource.
```

Comment:
```suggestion
Run Chef Infra Client periodically as a scheduled task on Microsoft Windows. Scheduled tasks provides visibility, configurability, and reliability around log rotation and permissions. You can configure the Chef Infra Client to run as a scheduled task using the [chef_client_scheduled_task](/resources/chef_client_scheduled_task/) resource.
```

Reply:
This sentence feels really awkward now.

Reply:
Could just be 

On Microsoft Windows, run Chef Infra Client periodically as a scheduled task.